build/js/src/triniti/sys/RedirectId.js

```diff
@@ -0,0 +1,51 @@
+import { Base64 } from 'js-base64';
+
+import Identifier from '@gdbots/pbj/well-known/Identifier';
+import NodeRef from '@gdbots/schemas/gdbots/ncr/NodeRef';
+import RedirectV1Mixin from './mixin/redirect/RedirectV1Mixin';
+
+/** @type {SchemaQName} */
+let qname;
+
+
+const urlsafeB64Encode = input => Base64.encode(input)
+  .replace(/\+/g, '-')
+  .replace(/\//g, '_')
+  .replace(/=+$/, '');
+
+/* eslint-disable no-mixed-operators */
+/* eslint-disable no-useless-escape */
+const urlsafeB64Decode = input => Base64.decode(
+  input + Array(5 - input.length % 4).join('=')
+    .replace(/\-/g, '+')
+    .replace(/\_/g, '/'));
+
+/**
+ * A redirect id is a friendly identifier.
```

Comment:
@hunyoboy make this comment match the php version, it's definitely not a friendly identifier